packages/reports/tests/acceptance/multi-datasource-test.js

```diff
@@ -45,9 +45,9 @@ module('Acceptance | multi-datasource report builder', function(hooks) {
 
     await selectChoose('.filter-builder__select-trigger', 'Between');
     await clickTrigger('.filter-values--dimension-date-range-input__low-value .ember-basic-dropdown-trigger');
-    await click($('button.ember-power-calendar-day--current-month:contains(4)')[0]);
+    await click('.ember-power-calendar-day--current-month:contains(4)');
```

Comment:
Does this selector work without jquery?

Reply:
Forgot to double check that, it doesnt work, what would you suggest? we were getting an error when using jquery in this file